Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -1500,64 +898,14 @@ public async Task PreloadIgnorePeerWithDifferentGenesisBlock()
             }
         }
 
-        [Fact(Timeout = Timeout)]
-        public async Task ReorgWhilePreloadAsync()
-        {
-            var seed = CreateSwarm();
-            var receiver = CreateSwarm();
-
-            BlockChain<DumbAction> seedChain = seed.BlockChain;
-            BlockChain<DumbAction> receiverChain = receiver.BlockChain;
-            for (int i = 0; i < 10; i++)
-            {
-                await seedChain.MineBlock(seed.Address);
-            }
-
-            bool reorg = true;
-
-            var progress = new ActionProgress<PreloadState>(state =>
-            {
-                _logger.Information("Received a progress event: {@State}", state);
-
-                if (reorg
-                    && state is BlockDownloadState blockDownloadState
-                    && blockDownloadState.ReceivedBlockHash.Equals(seedChain.Tip.Hash))
-                {
-                    var forked = seedChain.Fork(seedChain[9].Hash);
-                    seedChain.Swap(forked, false);
-                    seedChain.MineBlock(seed.Address).Wait();
-                    seedChain.MineBlock(seed.Address).Wait();
-
-                    reorg = false;
-                }
-            });
-
-            seed.Options.RecentStateRecvTimeout = TimeSpan.FromDays(1);
-
-            try
-            {
-                await StartAsync(seed);
-                await BootstrapAsync(receiver, seed.AsPeer);
-                await receiver.PreloadAsync(
-                    progress: progress,
-                    trustedStateValidators: new[] { seed.Address }.ToImmutableHashSet());
-
-                Assert.Equal(seedChain.Tip, receiverChain.Tip);
-            }
-            finally
-            {
-                await StopAsync(seed);
-            }
-        }
```

Comment:
I removed this test. The test checked it retries to get blocks again if the peer has reorg and has no states related to block hashes received in previous states.